stable/appmesh-controller/templates/webhook.yaml

```diff
@@ -2,11 +2,7 @@
 {{ $fullName := ( include "appmesh-controller.fullname" . ) }}
 {{ $webhookConfig := .Files.Get "webhookconfig.yaml" | fromYaml }}
 ---
-{{- if .Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1" }}
-apiVersion: admissionregistration.k8s.io/v1
-{{- else }}
 apiVersion: admissionregistration.k8s.io/v1beta1
-{{- end }}
```

Comment:
Seems like this change was directly merged into eks-charts and now this automated PR is removing it/overriding. Shouldn't this be included in the charts source code?